apps/files_sharing/src/views/SharingTab.vue

```diff
@@ -25,50 +25,124 @@
 				</SharingEntrySimple>
 			</ul>
 
-			<!-- add new share input -->
-			<SharingInput v-if="!loading"
-				:can-reshare="canReshare"
-				:file-info="fileInfo"
-				:link-shares="linkShares"
-				:reshare="reshare"
-				:shares="shares"
-				@open-sharing-details="toggleShareDetailsView" />
-
-			<!-- link shares list -->
-			<SharingLinkList v-if="!loading"
-				ref="linkShareList"
-				:can-reshare="canReshare"
-				:file-info="fileInfo"
-				:shares="linkShares"
-				@open-sharing-details="toggleShareDetailsView" />
-
-			<!-- other shares list -->
-			<SharingList v-if="!loading"
-				ref="shareList"
-				:shares="shares"
-				:file-info="fileInfo"
-				@open-sharing-details="toggleShareDetailsView" />
-
-			<!-- inherited shares -->
-			<SharingInherited v-if="canReshare && !loading" :file-info="fileInfo" />
-
-			<!-- internal link copy -->
-			<SharingEntryInternal :file-info="fileInfo" />
-
-			<!-- projects -->
-			<CollectionList v-if="projectsEnabled && fileInfo"
-				:id="`${fileInfo.id}`"
-				type="file"
-				:name="fileInfo.name" />
-		</div>
-
-		<!-- additional entries, use it with cautious -->
-		<div v-for="(section, index) in sections"
-			v-show="!showSharingDetailsView"
-			:ref="'section-' + index"
-			:key="index"
-			class="sharingTab__additionalContent">
-			<component :is="section($refs['section-'+index], fileInfo)" :file-info="fileInfo" />
+			<section>
+				<div class="section-header">
+					<h4>{{ t('files_sharing', 'Internal shares') }}</h4>
+					<NcButton v-tooltip="{
+							content: internalSharesHelpText,
+							triggers: ['hover'],
+							delay: { show: 0, hide: 400 },
+						}"
+						class="hint-icon"
+						type="tertiary-no-background"
+						:aria-label="t('files_sharing', 'Internal shares explanation')"
+						:aria-describedby="'tooltip-internal-shares'">
+						<span id="tooltip-internal-shares" class="visually-hidden">
+							{{ internalSharesHelpText }}
+						</span>
+						<template #icon>
+							<InfoIcon :size="20" />
+						</template>
+					</NcButton>
```

Comment:
Depending on your screen reader this still will be announced as:

> Button - Internal shares explanation

No further information, users with screen readers will then click on it and will not get anything else.
Thats why I recommend to implement it like this:

```suggestion
						<p class="hidden-visually">
							{{ internalSharesHelpText }}
						</p>
						<InfoIcon class="hint-icon"
							:size="20"
							:title="internalSharesHelpText" />
```

Reply:
```suggestion
				<NcPopover :focus-trap="false">
					<template #trigger>
						<NcButton :aria-label="t('files_sharing', 'Internal shares explanation')">
							<template #icon>
								<InfoIcon :size="20" />
							</template>
						</NcButton>
					</template>
					{{ internalSharesHelpText }}
				</NcPopover>
```